<!DOCTYPE html>
<html lang="en">
<head>
	<title>David Lougheed | Bioinformatics Full Stack Developer</title>

	<link rel="stylesheet" href="style.css">
	<link rel="shortcut icon" href="favicon.png">

	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta charset="utf-8">
</head>
<body>
<main>
	<header>
		<h1 id="leading">David Lougheed</h1>
		<ul id="nav">
			<li><a href="#publications">Publications</a></li>
			<li class="after-line"><a href="#software-projects">Software &amp; Projects</a></li>
			<li><a href="https://github.com/davidlougheed">GitHub</a></li>
			<li><a href="https://scholar.google.com/citations?user=dUVIzNwAAAAJ&hl=en">Google Scholar</a></li>
		</ul>
	</header>
	<p>
		<img alt="" src="img/me.jpeg" style="float: right; height: 220px; margin-left: 16px; width: auto;" id="pfp" />
		I am a software developer at the
		<a href="https://www.computationalgenomics.ca/">Canadian Centre for Computational Genomics (C3G)</a>
		in Montr&eacute;al. There, I am creating web portals for health and &#x2011;omics data, and developing a tool to
		<a href="#strkit">better genotype short tandem DNA repeats</a> using long-read sequencing data.
	</p>
	<p>
		I also occasionally work at the <a href="https://qubs.ca/">Queen's University Biological Station</a>, where I
		act as a field/network technician, helping to maintain networking infrastructure and a network of
		satellite-linked climate stations.
	</p>
	<p>
		I have a B.Sc. in Computer Science and Biology (graduated 2019 with First Class
		Honours), and an M.Sc. in Human Genetics (granted 2023); both from McGill University.
		My Master's thesis was supervised by Professor Guillaume Bourque, and is titled
		<a href="https://escholarship.mcgill.ca/concern/theses/jq085r38f">
			&ldquo;Using High-Fidelity Long-Read Sequencing to Better Detect and Understand Short Tandem Repeat
			Variation in Humans.&rdquo;</a>
	</p>

	<h2 id="publications">Publications</h2>
	<ul class="projects">
		<li>
			<a href="https://www.biorxiv.org/content/10.1101/2024.10.27.620512v1.full.pdf+html">
				Insights on macrosynteny, ′rebel′ genes, and a new sex-linked region in anurans from comparative
				genomics and a new chromosome-level genome for the western chorus frog
			</a> (2024)
			<p>
				<a class="subtle-link" href="https://yingchen94.github.io/">Ying Chen</a>,
				<strong>David R Lougheed</strong>, Zhengxin Sun, Jeffrey Ethier,
				<a class="subtle-link" href="https://www.teamendo.ca/">Vance L Trudeau</a>,
				<a class="subtle-link" href="https://sclougheed.ca">Stephen C Lougheed</a>.
				<em>BioRxiv preprint.</em>
				DOI: <a href="https://doi.org/10.1101/2024.10.27.620512">10.1101/2024.10.27.620512</a>
			</p>
		</li>
		<li>
			<a href="https://onlinelibrary.wiley.com/doi/10.1002/edn3.590">
				Enhancing metabarcoding of freshwater biotic communities: a new online tool for primer selection and
				exploring data from 14 primer pairs
			</a> (2024)
			<p>
				<a class="subtle-link" href="https://oriannetournayre.wixsite.com/website">Orianne Tournayre</a>,
				<a class="subtle-link" href="https://allensgallery.ca/">Haolun Tian</a>,
				<strong>David R Lougheed</strong>, Matthew JS Windle, Sheldon Lambert,
				Jennipher Carter, Zhengxin Sun, Jeff Ridal, Yuxiang Wang, Brian F Cumming,
				<a class="subtle-link" href="http://www.arnottlab.ca/">Shelley E Arnott</a>,
				<a class="subtle-link" href="https://sclougheed.ca">Stephen C Lougheed</a>.
				<em>Environmental DNA.</em>
				DOI:&nbsp;<a href="https://doi.org/10.1002/edn3.590">10.1002/edn3.590</a>
			</p>
		</li>
		<li>
			<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btae136/7623587">
				EpiVar Browser: advanced exploration of epigenomics data under controlled access</a> (2024)
			<p>
				<strong>David R Lougheed</strong>, Hanshi Liu, Katherine A Aracena, Romain Gr&eacute;goire,
				Alain Pacis, Tomi Pastinen,
				<a class="subtle-link" href="http://luis-barreirolab.org/">Luis B Barreiro</a>, Yann Joly, David Bujold,
				<a class="subtle-link" href="https://computationalgenomics.ca/BourqueLab/">Guillaume Bourque</a>.
				<em>Bioinformatics.</em>
				DOI:&nbsp;<a href="https://doi.org/10.1093/bioinformatics/btae136">10.1093/bioinformatics/btae136</a>
			</p>
		</li>
		<li>
			<a href="https://www.nature.com/articles/s41588-024-01668-z">
				Epigenetic variation impacts individual differences in the transcriptional
				response to influenza infection</a> (2024)
			<p>
				Katherine A Aracena, Yen-Lung Lin, Kaixuan Luo, Alain Pacis, Saideep Gona, Zepeng Mu,
				Vania Yotova, Renata Sindeaux, Albena Pramatarova, Marie-Michelle Simon, Xun Chen,
				Cristian Groza, <strong>David Lougheed</strong>, Romain Gr&eacute;goire, David Brownlee, Yang Li,
				Xin He, David Bujold, Tomi Pastinen,
				<a class="subtle-link" href="https://computationalgenomics.ca/BourqueLab/">Guillaume Bourque</a>,
				<a class="subtle-link" href="http://luis-barreirolab.org/">Luis B Barreiro</a>.
				<em>Nature Genetics.</em>
				DOI:&nbsp;<a href="https://doi.org/10.1038/s41588-024-01668-z">10.1038/s41588-024-01668-z</a>
			</p>
		</li>
		<li>
			<a href="https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13405">
				<em>baRcodeR</em>: An open‐source R package for sample labelling</a> (2020)
			<p>
				Yihan Wu, <strong>David R Lougheed</strong>,
				<a class="subtle-link" href="https://sclougheed.ca">Stephen C Lougheed</a>, Kristy Moniz,
				<a class="subtle-link" href="https://walkerlab.carrd.co/">Virginia K Walker</a>,
				<a class="subtle-link" href="https://www.ecoevogeno.org/">Robert I Colautti</a>.
				<em>Methods in Ecology and Evolution.</em>
				DOI:&nbsp;<a href="https://doi.org/10.1111/2041-210X.13405">10.1111/2041-210X.13405</a>
			</p>
		</li>
		<li>
			<a href="https://www.nature.com/articles/s41467-019-12829-8">
				Genome-wide microhomologies enable precise template-free editing of biologically
				relevant deletion mutations</a> (2019)
			<p>
				Janin Grajcarek, <a class="subtle-link" href="https://jmonlong.github.io/">Jean Monlong</a>,
				Yoko Nishinaka-Arai, Michiko Nakamura, Miki Nagai, Shiori Matsuo, <strong>David Lougheed</strong>,
				Hidetoshi Sakurai, Megumu K. Saito,
				<a class="subtle-link" href="https://computationalgenomics.ca/BourqueLab/">Guillaume Bourque</a> &amp;
				<a class="subtle-link" href="https://woltjen.cira.kyoto-u.ac.jp/">Knut Woltjen</a>.
				<em>Nature Communications.</em>
				DOI:&nbsp;<a href="https://doi.org/10.1038/s41467-019-12829-8">10.1038/s41467-019-12829-8</a>
			</p>
		</li>
	</ul>

	<h2 id="software-projects">Software &amp; Projects</h2>
	<ul class="projects">
		<li class="with-logo">
			<div>
				<a href="https://snipe.dlougheed.com/">SNIPe</a>
				<p>
					SNIPe is a web tool for optimally choosing environmental DNA (eDNA) primer pairs to identify a set
					of taxa. Part of a publication by
					<a href="https://onlinelibrary.wiley.com/doi/10.1002/edn3.590">Tournayre <em>et al.</em> (2024)</a>.
				</p>
			</div>
			<img alt="" src="img/snipe.svg" />
		</li>
		<li class="with-logo" id="strkit">
			<div>
			<a href="https://github.com/davidlougheed/strkit">STRkit</a>
			<p>
				STRkit is a <a href="https://en.wikipedia.org/wiki/Microsatellite">short tandem repeat (STR)</a>
				genotyper for long read sequencing data such as PacBio HiFi. STRkit was created as part of my Master's
				degree in Human Genetics at McGill.
			</p>
			</div>
			<img alt="" src="img/strkit.png" />
		</li>
		<li>
			<a href="https://flu-infection.vhost38.genap.ca/">EpiVar Browser</a>
			<p>
				A browser for exploring quantitative trait loci and their associations with
				genotype and ancestry. Published as a <em>Bioinformatics</em> application note:
				<a href="https://doi.org/10.1093/bioinformatics/btae136">Lougheed <em>et al.</em> (2024)</a>, and as part
				of a publication by <a href="https://doi.org/10.1038/s41588-024-01668-z">Aracena <em>et al.</em> (2024)</a>.
			</p>
		</li>
		<li>
			<a href="https://github.com/bento-platform/">Bento</a>
			<p>
				A platform for storing, analyzing, and visualizing genomic and epigenomic data,
				created while working at the
				<a href="https://www.computationalgenomics.ca/">Canadian Centre for Computational Genomics (C3G)</a>.
			</p>
		</li>
		<li class="with-logo">
			<div>
			David's Trail Guide System
			<p>
				A set of three components for creating mobile trail guide applications: a
				<a href="https://github.com/davidlougheed/trail-guide-content-server">content server</a> with a
				<a href="https://github.com/davidlougheed/trail-guide-content-web">corresponding front-end</a> and a
				<a href="https://github.com/davidlougheed/trail-guide-app">white-label Expo application</a>. I used this
				to build the latest version of the
				<a href="https://elbowlakecentre.ca/trail-guide/">Elbow Lake Trail Guide</a>.
			</p>
			</div>
			<img alt="" src="img/dtgs.png" />
		</li>
		<li>
			<a href="https://mhcut-browser.genap.ca">MHCut Browser</a> (2019)
			<p>
				A browser for microhomologies at variant CRISPR/Cas9 cut sites. Part of a publication
				by <a href="https://doi.org/10.1038/s41467-019-12829-8">Grajcarek <em>et al.</em> (2019)</a>.
			</p>
		</li>
		<li>
			<a href="https://elbowlakecentre.ca/trail-guide/">Elbow Lake Trail Guide</a>
			<p>
				An interactive trail guide app available on Android and iOS.
			</p>
		</li>
		<li>
			<a href="https://fieldstations.ca/">Canadian Field Stations</a> (2014-2016; 2024)
			<p>
				A compendium of field stations across Canada which I assist in maintaining.
			</p>
		</li>
	</ul>
</main>
</body>
</html>
