<!DOCTYPE html>
<html lang="en">
<head>
	<title>David Lougheed | Software &amp; Projects</title>

	<link rel="stylesheet" href="style.css">
	<link rel="shortcut icon" href="favicon.png">

	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta charset="utf-8">
</head>
<body>
<main>
	<header>
		<h1 id="leading">David Lougheed</h1>
		<ul id="nav">
			<li><a href="index.html">Home</a></li>
			<li><a href="publications.html">Publications</a></li>
			<li>Software &amp; Projects</li>
		</ul>
	</header>

	<h2 id="software-projects">Software &amp; Projects</h2>
	<ul class="projects">
		<li class="with-logo">
			<div>
				<a href="https://snipe.dlougheed.com/">SNIPe</a>
				<p>
					SNIPe is a web tool for optimally choosing environmental DNA (eDNA) primer pairs to identify a set
					of taxa. Part of a publication by
					<a href="https://onlinelibrary.wiley.com/doi/10.1002/edn3.590">Tournayre <em>et al.</em> (2024)</a>.
				</p>
			</div>
			<img alt="" src="img/snipe.svg" />
		</li>
		<li class="with-logo" id="strkit">
			<div>
				<a href="https://github.com/davidlougheed/strkit">STRkit</a>
				<p>
					STRkit is a <a href="https://en.wikipedia.org/wiki/Microsatellite">short tandem repeat (STR)</a>
					genotyper for long read sequencing data such as PacBio HiFi. STRkit was created as part of my Master's
					degree in Human Genetics at McGill.
				</p>
			</div>
			<img alt="" src="img/strkit.png" />
		</li>
		<li>
			<a href="https://flu-infection.vhost38.genap.ca/">EpiVar Browser</a>
			<p>
				A browser for exploring quantitative trait loci and their associations with
				genotype and ancestry. Published as a <em>Bioinformatics</em> application note:
				<a href="https://doi.org/10.1093/bioinformatics/btae136">Lougheed <em>et al.</em> (2024)</a>, and as part
				of a publication by <a href="https://doi.org/10.1038/s41588-024-01668-z">Aracena <em>et al.</em> (2024)</a>.
			</p>
		</li>
		<li class="with-logo">
			<div>
				<a href="https://github.com/bento-platform/">Bento</a>
				<p>
					A platform for storing, analyzing, and visualizing genomic and epigenomic data,
					created while working at the
					<a href="https://www.computationalgenomics.ca/">Canadian Centre for Computational Genomics (C3G)</a>.
				</p>
			</div>
			<img alt="" src="img/bento.png" />
		</li>
		<li class="with-logo">
			<div>
				David's Trail Guide System
				<p>
					A set of three components for creating mobile trail guide applications: a
					<a href="https://github.com/davidlougheed/trail-guide-content-server">content server</a> with a
					<a href="https://github.com/davidlougheed/trail-guide-content-web">corresponding front-end</a> and a
					<a href="https://github.com/davidlougheed/trail-guide-app">white-label Expo application</a>. I used this
					to build the latest version of the
					<a href="https://elbowlakecentre.ca/trail-guide/">Elbow Lake Trail Guide</a>.
				</p>
			</div>
			<img alt="" src="img/dtgs.png" />
		</li>
		<li>
			<a href="https://mhcut-browser.genap.ca">MHCut Browser</a> (2019)
			<p>
				A browser for microhomologies at variant CRISPR/Cas9 cut sites. Part of a publication
				by <a href="https://doi.org/10.1038/s41467-019-12829-8">Grajcarek <em>et al.</em> (2019)</a>.
			</p>
		</li>
		<li>
			<a href="https://elbowlakecentre.ca/trail-guide/">Elbow Lake Trail Guide</a>
			<p>
				An interactive trail guide app available on Android and iOS.
			</p>
		</li>
		<li>
			<a href="https://fieldstations.ca/">Canadian Field Stations</a> (2014-2016; 2024)
			<p>
				A compendium of field stations across Canada which I assist in maintaining.
			</p>
		</li>
	</ul>
</main>
</body>
</html>
